modules/phase_field/src/ics/MultiSmoothSuperellipsoidIC.C

```diff
@@ -156,13 +173,12 @@ MultiSmoothSuperellipsoidIC::computeSuperellipsoidCenters()
           rr = tmp_rr;
       }
 
-      if (i == 0) rr = _range.norm();
+      if (i == start) rr = _range.norm();
```

Comment:
Yeah, let's just redo this from scratch. The loop has two goals, right?
Enforcing a minimum desistance and preventing overlap. Those goals are
somewhat similar.

On Thu, Jul 7, 2016, 11:30 AM Floyd <notifications@github.com> wrote:

> In modules/phase_field/src/ics/MultiSmoothSuperellipsoidIC.C
> <https://github.com/idaholab/moose/pull/7282#discussion_r69950108>:
>
> > @@ -156,13 +173,12 @@ MultiSmoothSuperellipsoidIC::computeSuperellipsoidCenters()
> >            rr = tmp_rr;
> >        }
> >
> > -      if (i == 0) rr = _range.norm();
> > +      if (i == start) rr = _range.norm();
>
> This change will result in the while loop being executed unitl num_tries
> equals _max_num_tries because rr will never change from 0.0.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/idaholab/moose/pull/7282/files/7ef908a574dd2485c807704bb64b36a466068705#r69950108>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAMWPvqRfPmKpCkwYChOVPk_W5_b3e7Rks5qTTfSgaJpZM4JGSPX>
> .
>